### DISCUSSION TITLE
[ToggleGroup] Example with type="multiple"

### DISCUSSION THREAD

**Discussion by taismassaro** [radix-ui/primitives#1768] [Help]
### ToggleGroup

Hi there, I'm starting to work on a rich text editor for a project and would love to use RadixUI for the first time!
The main use case for the POC I'm working on is buttons for Bold and Italic, for example. So I would like to use the ToggleGroup component with the ability to select multiple items, but I'm struggling to figure out how to do that with a single `onValueChange` on the Root component. Would be nice to have some example provided in the docs on how to use this with `type="multiple"`.

Thanks a lot!

**Comment by benoitgrelard** [comment]
Hey, yeah we should add an example, in the meantime though, it's pretty simple. When you pass `type="multiple"` the `value`/`defaultValue` and `onValueChange` will now all work off an array of strings instead of a single string.

**Reply by taismassaro** [reply]
hey @benoitgrelard 
thanks for the quick response!
I got that difference in the type of the value, but I'm working with toggle functions from https://github.com/ueberdosis/tiptap (basically a toggle function for each item/button based on the editor's internal state) and trying to figure out how to manage that with only one `onValueChange` on the group. do you have any thoughts or suggestions?

**Reply by benoitgrelard** [reply]
Hey Taís, I don't know what those are, but if you create a barebones codesandbox with these 2 together, I can try and help?
